ci-operator/step-registry/windows/e2e/operator/test/upgrade/upgrade/windows-e2e-operator-test-upgrade-upgrade-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: windows-e2e-operator-test-upgrade-upgrade
+  cli: latest
+  from: operator-sdk
+  commands: windows-e2e-operator-test-upgrade-upgrade-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 400Mi
+  dependencies:
+    - env: OO_BUNDLE
```

Comment:
OO_bundle-> wmco-bundle

OO_BUNDLE is a holdover from OO_INDEX.
it can be renamed to something that makes more sense.

Reply:
I agree, and this falls beyond the scope of generalizing the existing logic.

Reply:
I disagree. The envvar `OO_BUNDLE` has no meaning within this workflow. Because you are adding this as a general step registry step it needs to make sense on its own.

Reply:
It is no longer applicable since PR was reframed as "Move WMCO bundle upgrade testing to Azure."